drivers/Makefile.dep

```diff
@@ -18,6 +18,19 @@ ifneq (,$(filter at86rf2%,$(USEMODULE)))
   endif
 endif
 
+ifneq (,$(filter mrf24j40,$(USEMODULE)))
+  USEMODULE += mrf24j40
```

Comment:
I think that one is directly copied from the at86rf2% above it. While it makes sense there, it looks indeed useless here.